_studio/shared/src/libmfx_core_vaapi.cpp

```diff
@@ -259,6 +259,8 @@ typedef struct {
     { 0x3EA6, MFX_HW_CFL, MFX_GT3 },
     { 0x3EA7, MFX_HW_CFL, MFX_GT3 },
     { 0x3EA8, MFX_HW_CFL, MFX_GT3 },
+    { 0x3E9C, MFX_HW_CFL, MFX_GT1 },
+    { 0x3E99, MFX_HW_CFL, MFX_GT1 },
```

Comment:
And this one too.

Reply:
File a bug/PR against media-driver to add this ID. We can't merge in this change before media-driver will not be updated.